node-sdk-sample/package.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "sdk-sample",
+  "version": "1.0.0",
+  "main": "index.js",
+  "type": "commonjs",
+  "directories": {
+    "test": "tests"
+  },
+  "scripts": {
+    "test": "jest",
+    "lint": "eslint src",
+    "build": "tsc"
+  },
+  "author": "",
+  "license": "ISC",
+  "description": "",
```

Comment:
I think it would be good to populate these fields with author = Zowe, license = EPL-2.0, and description = Something that sounds nice 😋